packages/utils/src/saml/bulk-saml.ts

```diff
@@ -0,0 +1,64 @@
+// This is a helper script that lets you test constructing your own soap+saml requests. It creates the SOAP Envelope and then sends it to the gateway specified in the request body.
+// It logs the output into the runs/saml-coverage folder where it can be analyzed later. Reccomendation is to use the saml-coverage script to analyze the results after running.
+// npm run saml-server and then reference the Metriport- IHE GW / XML + SAML Constructor - Postman collection
```

Comment:
nit: please, let's keep the comments well taken care of like we do for the rest of the code - [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#493564ff82fa45edb294b6a729e94fba):
- max column length is 100 chars
- multi-line comments use `/** */`
- top-level comments go after the import (save pre-import to basic file header, like license)